components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -386,7 +389,7 @@ - (void)viewWillLayoutSubviews {
     }
   } else {
     contentViewFrame.size.height = self.contentViewController.preferredContentSize.height;
-    if ([self isAccessibilityMode]) {
+    if ([self isAccessibilityMode] || [self isMobileLandscape]) {
```

Comment:
_Nit:_ This combined `if` condition appears 4 times in here, consider combining them into something like `shouldAlwaysPresentFullscreen`

Reply:
good feedback! done.